### ISSUE TITLE
Cart images aren't always loading

### ISSUE THREAD

**Issue by erinemay** [iFixit/react-commerce#984]
Repro steps: ??
I tried on my own devices:
- Windows 10 + Chrome and FF
- MacOS Big Sur + Safari
- Pixel + Chrome

I tried a handful of products/states:
- Common products (from the first couple of collection pages) and uncommon ones, like those on https://www.ifixit.com/Parts?p=52
- Adding to an empty and non-empty cart
- Logged in and logged out

Sometimes the image loads a little slower than the cart, and I can see a split second of no image before it loads. This is exacerbated when I toggle my internet all the way down, but I haven't been able to repro the non loading.

From @danielbeardsley's slack comment, this cart product image isn't loading when the cart drawer opens directly after adding to cart:
https://www.ifixit.com/products/macbook-pro-a1989-a1990-a2141-a2159-a2251-a2289-a2338-mid-2018-2020-usb-c-board
![image](https://user-images.githubusercontent.com/72166715/200020706-1571464b-3fde-4c47-97d2-184cf74c991f.png)

@danielbeardsley, @sterlinghirsh, what do you two have in common? You've both experienced this. Slack: [Danny](https://ifixit.slack.com/archives/C01K32B4FPX/p1667523766710619?thread_ts=1667522653.123449&cid=C01K32B4FPX), [Sterling](https://ifixit.slack.com/archives/C03TRK8SE4A/p1667405319943989).

**Comment by danielbeardsley** [comment]
Well... it's not necessarily a bug in `react-commerce` cause the cart API returns the "no-image" state when the product is in my cart:

After adding it to my cart, I visit:
https://www.ifixit.com/api/2.0/store/user/cart
![image](https://user-images.githubusercontent.com/26855/200026248-40a0ff7e-0f3f-42e7-8e98-baea5ffd40fc.png)

The cart drawer mostly just shows the data from this API

**Comment by danielcliu-ifixit** [comment]
![Screenshot from 2022-11-04 11-02-59](https://user-images.githubusercontent.com/109113415/200044884-27149260-0f28-4047-95fa-d6ac12fb69f8.png)
Hmm not sure why danny gets a noImageFixie and i dont

**Comment by danielbeardsley** [comment]
> Hmm not sure why danny gets a noImageFixie and i dont

Check your api response from: https://www.ifixit.com/api/2.0/store/user/cart

**Comment by danielcliu-ifixit** [comment]
![Screenshot from 2022-11-04 13-03-55](https://user-images.githubusercontent.com/109113415/200064819-315ba5bd-8d50-4f4c-a358-35c516cd677e.png)

**Comment by danielbeardsley** [comment]
Figured it out! It has to do with my store preference. When set to non-US, I get missing images in that API.

**Comment by danielcliu-ifixit** [comment]
@erinemay could that be the heart of the issue ^

**Comment by erinemay** [comment]
Yep. Just reproed by viewing my react-commerce mini cart with a non-us store selected as my preferred store.

**Comment by danielbeardsley** [comment]
This is likely a bug in our php app FYI, so I'm moving the issue there.
